docs/api.md

```diff
@@ -456,17 +465,41 @@ Emitted when a target is created, for example when a new page is opened by [`win
 
 Emitted when a target is destroyed, for example when a page is closed.
 
+#### browser.browserContexts()
+- returns: <[Array]<[BrowserContext]>>
+
+Returns all the browser contexts created in the browser. In a newly created browser, this will return
```

Comment:
Returns an array of all open browser contexts. In a newly created browser, ...

Reply:
Done.